10_datastructures_lab/01.py

```diff
@@ -1,19 +1,8 @@
-"""
-Write a program that takes two strings
-from the user and checks if they represent
-a valid user name.
-Valid users and passwords:
-    apple => red
-    lettuce => green
-    lemon => yellow
-    orange => orange
-
-When valid credentials are entered print:
-    Welcome Master
-
-And when invalid credentials are entered print:
-    INTRUDER ALERT
-
-and exit the program with a non-zero exit code
-"""
-
+"""get name and password and check if it correct"""
+import os,sys
+nep = {'apple' : "red" , 'lettuce' : "green" , 'lemon' : "yellow" , 'orange' : "orange"}
+for key in nep:

```

Comment:
That's not the best way to use a dictionary-

Dictionaries are really good at searching values, so we can do `if key in dict` to get a true/false result if a key is in the dictionary, or `dict[key]` to read a value by a given key.

So in your case I'd recommend:

```
username, password = sys.argv[1:]

if username in nep:
  if password == nep[username]:
    print "welcome master"
```